applications/applications.c

```diff
@@ -17,6 +17,7 @@ extern int32_t desktop_srv(void* p);
 extern int32_t updater_srv(void* p);
 
 // Apps
+extern int32_t clock_app_old(void *p);
```

Comment:
Why is it "old", where's "new"?

Reply:
CompaqDisc was working on a version with Settings and Alarm